gnomad/utils/release.py

```diff
@@ -0,0 +1,94 @@
+from typing import Dict, List, Optional, Union
+
+import hail as hl
+
+from gnomad.resources.grch38.gnomad import (
+    GROUPS,
+    POPS,
+    SEXES,
+    SUBSETS,
+)
+from gnomad.utils.vcf import index_globals, make_label_combos
+
+
+def make_faf_index_dict(
+        faf_meta: List[Dict[str, str]],
+        groups: List[str] = ["adj"],
+        pops: List[str] = POPS,
+        sexes: List[str] = SEXES,
+        label_delimiter: str = "_",
+) -> Dict[str, int]:
+    """
+    Create a look-up Dictionary for entries contained in the filter allele frequency annotation array.
+
+    :param faf_meta: Global annotation containing the set of groupings for each element of the faf array
+        (e.g., [{'group': 'adj'}, {'group': 'adj', 'pop': 'nfe'}])
+    :param groups: List of sample groups [adj, raw]. Default is GROUPS.
+    :param pops: Dict of sample global population names for gnomAD genomes. Default is POPS.
```

Comment:
```suggestion
    :param pops: Dict of sample global population names for gnomAD genomes. Default is POPS
```

Reply:
This is driving me nuts, but I fixed this in my local branch and pushed it, but it's not showing up in the GitHub UI for some reason